github/pkg/apis/sources/v1alpha1/githubsource_types_test.go

```diff
@@ -285,13 +329,32 @@ func TestGitHubSourceStatusGetCondition(t *testing.T) {
 			Status: corev1.ConditionTrue,
 		},
 	}, {
-		name: "mark sink empty, secrets, and event types",
+		name: "mark sink, secrets, and event types, then no webhook",
+		s: func() *GitHubSourceStatus {
+			s := &GitHubSourceStatus{}
+			s.InitializeConditions()
+			s.MarkSink("uri://example")
+			s.MarkSecrets()
+			s.MarkEventTypes()
+			s.MarkNoWebhook("Testing", "hi%s", "")
+			return s
+		}(),
+		condQuery: GitHubSourceConditionReady,
+		want: &apis.Condition{
+			Type:   GitHubSourceConditionReady,
+			Status: corev1.ConditionFalse,
```

Comment:
<!--gotool-bot-->
Format Go code:
```suggestion
			Type:    GitHubSourceConditionReady,
			Status:  corev1.ConditionFalse,
```

Reply:
Thanks! It has been done and pushed. Does it look good?